src/components/select-field/SelectField.scss

```diff
@@ -12,6 +12,23 @@
     }
 }
 
+.search-input-container {
+    width: 100%;
+    padding-right: $bdl-grid-unit;
```

Comment:
Think about why I asked for using the grid. We can use SASS to get multiples of the grid unit so all measures are relative to the base unit. `padding-bottom: $bdl-grid-unit * 2`

Reply:
Thanks for clarifying!